### DISCUSSION TITLE
Missing play_started callback in iOS SDK when play request is made before the streaming starts

### DISCUSSION THREAD

**Discussion by Mohit-3196** [ant-media/Ant-Media-Server#5248] [FAQs]
We are using the Ant Media Server iOS SDK for live streaming. Our scenario involves publishing streams at a defined time. However, we have noticed that if a user tries to play a stream before it has started, the playback does not begin instantly. and even after the stream has started, we are not receiving the `play_started` callback. As a result, the user is unable to play the stream.

If the user attempts to play the stream after it has started, everything works fine.

We would like to understand why the `play_started` callback is not triggered when playing a stream before it starts, and if there is any solution or workaround to ensure the callback is received once the stream begins.

Thank you

**Comment by mekya** [comment]
Thank you for the question. Yes, I've confirmed the case and I've created [this issue](https://github.com/ant-media/Ant-Media-Server/issues/5258) 

Fortunately, I've also sent a fix for this issue [in this PR.](https://github.com/ant-media/WebRTC-iOS-SDK/pull/24)

You can just import the branch as an SPM dependency and test it on your end.
